src/main/java/ch/heigvd/dai/commands/Root.java

```diff
@@ -196,10 +209,18 @@ public String getText() {
     return text;
   }
 
+  public AvailableInputFormat getInputFormat() {
+    return inputFormat;
+  }
+
   public boolean isShow() {
     return show;
   }
 
+  public boolean isVerbose() {
```

Comment:
There are a lot of methods like this that are practically empty. I'm wondering if we could remove them, that might improve the readability.

Reply:
In our case, getters and setters are not really useful because we are in the same class.
In my opinion they can be removed.